### PR TITLE
VSCode workspace support/clean up package.json

### PR DESCRIPTION
**Pull request by smorrison**
For your consideration, I'm not insisting that we accept this.

The PR adds a vs-code workspace file. One of the conveniences is that it suggests recommended extensions to the editor for eslint support, etc. 

The other thing is I pulled the package.json info down from the sub-directories, tried to pry apart the dev dependencies from the production dependencies, then added scripts to the package.json to run things conveniently (ie. 'npm start' runs express, 'npm test' runs all tests)

### CODE REVIEW

**Comment by mvahowe** [comment]
Thanks @smorrison I'll try this tomorrow. Assuming that everything that worked before works now, I'd be inclined to accept this PR so we can all try working with it and see what we don't like about it.

**Comment by smorrison** [comment]
I rebased after merging the other pr (which added the import zip endpoints). There's got to be a better way to merge package.json.

File: package.json [hunk 1]
```diff
@@ -1,31 +1,71 @@
...
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-mocha": "^6.3.0",
+    "eslint-formatter-pretty": "^3.0.1",
+    "prettier": "^1.19.1",
+    "pretty-format": "^25.3.0",
+    "tape": "^4.13.2",
+    "mocha": "^7.1.1"
+  },
+  "eslintConfig": {
```
Comment (ericpyle) [inline package.json:58]: @smorrison maybe this eslintConfig info should move to test/eslintrc.json since it should only apply to tests?

**Comment by smorrison** [comment]
closed in favour of more atomic PRs

**Pull request closed without merging**